src/resolution/instantiation.ts

```diff
@@ -116,7 +115,7 @@ function _postConstruct<T>(constr: interfaces.Newable<T>, instance: T): void | P
 }
 
 function _validateInstanceResolution(binding: interfaces.Binding<unknown>, constr: interfaces.Newable<unknown>): void {
-    if (binding.scope === "Transient") {
+    if (binding.scope.type === ConfigurableBindingScopeEnum.Transient) {
```

Comment:
This will be changed to Singleton https://github.com/inversify/InversifyJS/pull/1335